tests/spawn-utils.c

```diff
@@ -69,6 +69,17 @@ cmd_result_ensure_ok (CmdResult *cmd,
 }
 
 gboolean
+cmd_result_ensure_ok_verbose (CmdResult *cmd)
```

Comment:
Nitpick: `cmd_result_ensure_ok_verbose()` is now fairly nonsensical as a function.